models/issues/review.go

```diff
@@ -615,12 +615,14 @@ func AddReviewRequest(ctx context.Context, issue *Issue, reviewer, doer *user_mo
 		Issue:           issue,
 		RemovedAssignee: false,       // Use RemovedAssignee as !isRequest
 		AssigneeID:      reviewer.ID, // Use AssigneeID as reviewer ID
-		ReviewID:        review.ID,
```

Comment:
I don't quite understand why you no longer set the review ID.
Not setting it looks much more like a problem to me than omitting it.

Reply:
we do else have a lot of foreign keys witch point to nothing ...
... we already do xorm is just not aware of such stuff

Reply:
?
And?

I don't understand your point.
What does review deletion have to do with review creation?

Reply:
its not about reviews it's about review **requests**

Reply:
Yes, but the review exists at the time of the review request creation?

Reply:
at the time of creating this comment it exists yes so we could still have it put in.

later one it wont exist. I can add this line back and add a todo to https://github.com/go-gitea/gitea/pull/28544 
to have it removed if the review request get's deleted

Reply:
I mean we can simply add a consistency check?

Reply:
Nevertheless, I **strongly** doubt removing this line will do anything good.

Reply:
I'll put  it back ... but it wont be of good use I can tell you anyway